corehq/apps/accounting/subscription_changes.py

```diff
@@ -6,7 +6,7 @@
 from django.utils.translation import ugettext_lazy as _, ungettext
 from corehq import privileges
 from corehq.apps.accounting.utils import get_active_reminders_by_domain_name
-from corehq.apps.app_manager.models import Application
+from corehq.apps.app_manager.models.common import Application
```

Comment:
Maybe you wanted to do this, but an easier approach to iterate on I'd think is to import Application etc. into `models/__init__.py` so that the external imports don't have to change. Otherwise it's problematic for anyone who adds an import of `Application` between now and when this is merged.